index.html

```diff
@@ -12446,11 +12446,47 @@ <h2>A project for first-time contributions</h2>
                             </li>
                         </ul>
                     </div>
-                </div>  
+                </div>
                 <!-- ________ Akshat's card END ________  -->
 
 
 
+                <div class="card">
```

Comment:
As per the instructions, you need to copy the START and END comment lines as well.
```
<!-- COPY everything BELOW this, from contributor card start to end along with the "START" and "END" comment lines -->
```

Please add these 2 lines to your code so that I can merge.